tutorials/algorithms/11_trotterQRTE.ipynb

```diff
@@ -14,30 +14,30 @@
    "id": "f52a038f-f875-48f2-8345-039933a1fa69",
    "metadata": {},
    "source": [
-    "As a real time evolution technique, the Trotterization or Pauli-Trotter evolution consists in the successive application of a quantum gate, assumed to approximate the time evolution of a system for a time slice. Following from the Schrödinger equation, the time evolution of a system initially in the state $\\psi(0)$ takes the form:\n",
+    "As a real time evolution technique, the Trotterization or Trotterized Real Time Evolution (RTE) consists in the successive application of a quantum gate, assumed to approximate the time evolution of a system for a time slice. Following from the Schrödinger equation, the time evolution of a system initially in the state $\\vert\\psi(0)\\rangle$ takes the form:\n",
     "\n",
     "$$\n",
-    "| \\psi(t) \\rangle = e^{-i H t} | \\psi(0) \\rangle \\text{,}\n",
+    "\\vert \\psi(t) \\rangle = e^{-i H t} \\vert \\psi(0) \\rangle \\text{,}\n",
     "$$\n",
     "\n",
-    "where $H$ is the time-independent Hamiltonian under which the system is submitted. Such hamiltonian typically consists of a sum of many terms, $H = \\sum_j H_j$. Thus, the gate that approximates a single step is found under the assumption that the product of matrix exponentials is a good approximation for the sum of matrix exponentials:\n",
+    "where $H$ is the time-independent Hamiltonian under which the system is submitted. Such Hamiltonian typically consists of a sum of many terms, $H = \\sum_j H_j$. Thus, the sequence of gates that approximate a single evolution step is found under the assumption that the product of matrix exponentials is a good approximation for the sum of matrix exponentials:\n",
```

Comment:
We consider a Hamiltonian that can be written as a weighted sum of Pauli terms $H = \\sum_j a_j h_j$ with $h_j$ representing a tensor product of Pauli terms acting on $n$ qubits.

Reply:
Ideally, I'd like to see a statement like [this](https://qiskit.org/documentation/stubs/qiskit.synthesis.LieTrotter.html). You can find a bit more detail on [Wikipedia](https://en.wikipedia.org/wiki/Lie_product_formula).